realm/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -4,3 +4,4 @@ distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-bin.zip
+set JAVA_OPTS="-Xmx512m"
```

Comment:
I'll remove it again then. I was trying to fix a Java Heap space bug when compiling